### PR TITLE
Feature/kind cluster init

### PR DESCRIPTION
**Pull request by dblane-digicatapult**
Adds init scripts for starting up kind and bootstrapping flux2
Adds dscp-nodes using identities Alice, Bob and Charlie
Adds Document for getting-started.

### CODE REVIEW

File: clusters/kind-cluster/base/namespaces.yaml [hunk 1]
```diff
@@ -4,4 +4,21 @@ metadata:
   labels:
     app.kubernetes.io/instance: dscp
     app.kubernetes.io/version: latest
-  name: dscp
+  name: alice
+---
```
Comment (n3op2) [inline clusters/kind-cluster/base/namespaces.yaml:8]: aren't you missing `...`  to indicate the end of a document?

File: clusters/kind-cluster/charlie/dscp-node/release.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      chart: dscp-node
+      version: "3.5.0"
+      sourceRef:
+        kind: HelmRepository
+        name: dscp-node
+  valuesFrom:
+  - kind: ConfigMap
+    name: charlie-values
+    valuesKey: values-dscp-node.yaml
+  interval: 10m0s
```
Comment (n3op2) [inline clusters/kind-cluster/charlie/dscp-node/release.yaml:22]: not sure if this is possible but would be a lot easier to read if it was just `10m`

File: scripts/install-flux.sh [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    local ret=$?
+    if [ "$ret" -eq 0 ]; then
+      printf "OK\nKustomization \"flux-system\" successfully installed in $context\n"
+    elif [ "$ret" -ne 0 ]; then
+        printf "ERROR\nKustomization \"flux-system\" failed to install in $context\n"
+        exit 1
+    fi
+}
+
+assert_command kubectl
```
Comment (n3op2) [inline scripts/install-flux.sh:169]: might be nice to wrap all of this into a `main()` or any other function.

**Review by n3op2 — DISMISSED** [review]
Liike the README update

File: clusters/kind-cluster/base/namespaces.yaml (hunk 1 above)
Comment (dblane-digicatapult) [reply clusters/kind-cluster/base/namespaces.yaml:8]: Nope.

File: scripts/add-kind-cluster.sh [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 3080
+    protocol: tcp
+  - containerPort: 443
+    hostPort: 3443
+    protocol: tcp
```
Comment (mattdean-digicatapult) [inline scripts/add-kind-cluster.sh:53]: We should forward the node ports here so we can access the node from the polkadot gui

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**